src/Contract/Ast/TaggedTokenReferenceInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Contract\Ast;
+
+/**
+ * Represents the AST-Token, its location, and associated tags.
+ */
+interface TaggedTokenReferenceInterface extends TokenReferenceInterface
+{
+    public function hasTag(string $name): bool;
+
+    /**
+     * @return string[]|null
+     */
+    public function getTagLines(string $name): ?array;
+
+    public function getTagText(string $name): ?string;
```

Comment:
This method does not seem to be used anywhere.

Reply:
Right... I was originally thinking that I wanted to do the regex match against the merged text, but then decided against it. I'll remove it.